packages/mssql/src/MssqlSink.ts

```
@@ -243,7 +243,14 @@ export class MssqlSink
                 database: this.config.database,
                 result: MssqlMetricResults.Error,
             });
-            await tx.rollback();
+            try {
+                await tx.rollback();
```

Should we do something like in example: https://github.com/tediousjs/node-mssql#transaction?  So that w know we should not rollback due to it already happening?

---

we could, though I can't think of a reason why we'd ever want to bubble up the rollback error if there was a previous error that is the reason why we are rolling back in the first place

---

Do we have any actions we can take when a transaction was unsuccessfully rolled back?

If we hide the success/fail of a rollback would that not hide relevant information from the developer and result in some unknown state?